### PR TITLE
Add UBERON:8410021 inguinal region skin

### PR DESCRIPTION
**Pull request by paolaroncaglia**
Addresses #1668

### CODE REVIEW

**Comment by paolaroncaglia** [comment]
@dosumis 
For your review please.

File: uberon_edit.obo [hunk 1]
```diff
@@ -195307,6 +195307,21 @@ intersection_of: connects UBERON:0002116 ! ileum
...
+def: "The skin covering the inguinal region between the abdomen and the thigh on either side of the pubic bone." [http://orcid.org/0000-0001-7505-5418, PMID:32327715, PMID:468709, WikipediaVersioned:Groin&oldid=998393002]
+synonym: "groin skin" EXACT []
+synonym: "inguinal skin" EXACT []
+synonym: "skin of groin" EXACT []
+synonym: "skin of inguinal region" EXACT []
+synonym: "zone of skin of groin" EXACT []
+synonym: "zone of skin of inguinal region" EXACT []
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0008337 ! inguinal part of abdomen
+property_value: added_for_HCA "" xsd:string
```
Comment (dosumis) [inline uberon_edit.obo:195322]: Seems to be some issue with subset tagging here. Guide here:  https://ontology-development-kit.readthedocs.io/en/latest/NewSlimTerm.html

**Review by dosumis — CHANGES_REQUESTED** [review]
Subset tag needs fixing.  Everything else is good.

File: uberon_edit.obo (hunk 1 above)
Comment (paolaroncaglia) [reply uberon_edit.obo:195322]: @dosumis 
Thanks, there may be more like that... I made a note to fix any other. Now there's a conflict...

**Comment by paolaroncaglia** [comment]
@dosumis 
conflict fixed

**Review by dosumis — APPROVED** [review]

**Pull request merged**